tests/Commands/DrupalCommandsTest.php

```diff
@@ -68,11 +68,62 @@ public function testPermissions(array $config, $command, $expected_error, $expec
         $this->assertEquals($expected_settings_file_permission, $siteSettingsPermissions);
     }
 
+    /**
+     * Test the services file setup.
+     *
+     * @param array $configs
+     * @param array $expected
+     *
+     * @dataProvider servicesSetupDataProvider
+     */
+    public function testServicesSetup(array $configs, array $expected)
+    {
+        $config = $configs['runner'];
+        $configFile = $this->getSandboxFilepath('runner.yml');
+        file_put_contents($configFile, $config);
+
+        // Process information for destination file services.yml.
+        $sites_subdir = isset($config['drupal']['site']['sites_subdir']) ? $config['drupal']['site']['sites_subdir'] : 'default';
+        $services_destination_dir = $this->getSandboxRoot() . '/sites/' . $sites_subdir;
+        $services_destination_file = $services_destination_dir . '/services.yml';
+
+        // Process information for force option.
+        $force = '';
+        if (isset($configs['force'])) {
+            mkdir($services_destination_dir, 0777, true);
+            file_put_contents($services_destination_file, 'parameters: false');
```

Comment:
We should have `'parameters: false'` as part of the fixture. If more comfortable you can create a separate test for the force option.

Reply:
Ok, it's in fixture now.